examples/xml/xml.go

```diff
@@ -28,16 +28,16 @@ func main() {
 	coffee := &Plant{Id: 27, Name: "Coffee"}
 	coffee.Origin = []string{"Ethiopia", "Brazil"}
 
-	// 传入我们声明了 XML 的 plant 类型。
+	// 传入我们声明了 XML 的 Plant 类型。
 	// 使用 `MarshalIndent` 生成可读性更好的输出结果。
 	out, _ := xml.MarshalIndent(coffee, " ", "  ")
 	fmt.Println(string(out))
 
 	// 明确的为输出结果添加一个通用的 XML 头部信息。
 	fmt.Println(xml.Header + string(out))
 
-	// 使用 `Unmarshal` 将 XML 格式的字节流解析到 struct 内。
-	// 如果 XML 格式不正确，或无法映射到 struct，将会返回一个描述性错误。
+	// 使用 `Unmarhshal` 将 `XML` 格式字节流解析到 `Plant` 结构。
```

Comment:
```suggestion
	// 使用 `Unmarshal` 将 `XML` 格式字节流解析到 `Plant` 结构。
```

Unmarhshal 应该是英文作者的拼写错误，这里的函数名是 `xml.Unmarshal()`

Reply:
确实是这样，从原网页拷过来翻译，没留意倒原作者写错了。。感谢修正！